ReleaseNotes-2.11

```diff
@@ -42,4 +42,5 @@ Bugfixes:
 Intentional changes:
 ====================
 
-
+  * Next features are not going to be enabled/disabled by configuration anymore and they will be now available for all users:
```

Comment:
The fact they they are not configured anymore is not really important. Only that it's there now by default. Something like

Features enabled by default:
- Image Templates

Reply:
done!